indexer/bag_indexer/src/main/scala/uk/ac/wellcome/platform/archive/indexer/bags/Main.scala

```diff
@@ -47,9 +47,16 @@ object Main extends WellcomeTypesafeApp {
       elasticClient = elasticClient
     )
 
+    val maxExpectedFilesOnABag = 1000000
```

Comment:
Why are you defining this constant if you don't use it below?

Reply:
FFS typo from addressing the last review comment.